api/src/model/request/play_asset_pack_fetch_request.gd

```diff
@@ -82,21 +82,9 @@ func get_error() -> PlayAssetPackException:
 # Callback functions handling signals emitted from the plugin.
 # -----------------------------------------------------------------------------
 func _on_fetch_success(result: Dictionary):
-	# Since fetch() in plugin returns a PlayAssetPackStates Dictionary, we need to extract
-	# the PlayAssetPackState within.
-	var fetch_asset_pack_states_dict = PlayAssetPackStates.new(result).get_pack_states()
-	
-	if fetch_asset_pack_states_dict.has(_pack_name):
-		_on_state_updated(fetch_asset_pack_states_dict[_pack_name].to_dict())
-	else:
```

Comment:
Since we are not using fetch_success anymore, I am considering remove this ```INTERNAL_ERROR``` handling as well as related test cases.